hibernate-ehcache/src/main/java/org/hibernate/cache/ehcache/ConfigSettings.java

```diff
@@ -12,8 +12,6 @@
  * @author Steve Ebersole
  */
 public interface ConfigSettings {
-	String SIMPLE_FACTORY_NAME = "jcache";
```

Comment:
@sebersole Nothing requires that `SIMPLE_FACTORY_NAME` to be defined in these interfaces right? No magic I could find, but… 

Reply:
@alexsnaps It is just the name under which we register the factory for users to reference in config.  We recommend users use "short naming" to avoid refactoring impacts.  

This one is particular though is just a bad copy/paste.  The Ehcache provider actually registers 2 factories (`ehcache` and `ehcache-singleton`) which we could add to this ConfigSettings.

The intention is for users to have a static reference, much like we do with the larger  {{AvailableSettings}}.

Reply:
Don't we need an "ehcache" constant somewhere?

Reply:
Define "need"

Reply:
See `org.hibernate.cache.ehcache.internal.StrategyRegistrationProviderImpl`

Reply:
@sebersole Yes. You answer arrived while I was typing. So all good. We said the same thing.